ofrak_core/ofrak/core/free_space.py

```diff
@@ -283,10 +283,27 @@ async def _allocate(
 
     @staticmethod
     def _align_range(unaligned_range: Range, alignment: int) -> Range:
+        """
+        Increase the range start address so it is a multiple of the `alignment` size
```

Comment:
Might want to make it more explicit (here and in the return value description) that the end address is NOT modified at all. Looking back at this method, that's probably the main difference from what someone would infer from the name and signature, and it's also the reason for the only error case.